oci/casext/refname.go

```diff
@@ -206,10 +198,8 @@ func (e Engine) ListReferences(ctx context.Context) ([]string, error) {
 
 	var refs []string
 	for _, descriptor := range index.Manifests {
-		ref, ok := descriptor.Annotations[ispec.AnnotationRefName]
-		if ok {
-			refs = append(refs, ref)
-		}
+		ref := descriptor.Digest.String()
```

Comment:
This will break a lot of existing code and doesn't seem generally necessary to accomplish the goal in the commit description?

Reply:
It's also semantically incorrect -- the point of this method is to give you the list of all reference names in the image. A list of digests (without blob type or any other important information) is not equivalent.